tpchgen-cli/src/main.rs

```diff
@@ -91,11 +91,11 @@ struct Cli {
     tables: Option<Vec<Table>>,
 
     /// Number of parts to generate (manual parallel generation)
-    #[arg(short, long, default_value_t = 1)]
+    #[arg(short, long, default_value_t = -1)]
```

Comment:
If I didn't make this change, then tpchgen-cli was always generating at most 16 parts...